comp/updater/localapi/component.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package localapi is the updater local api component.
+package localapi
```

Comment:
Why localapi, and not just api? It seems that we already use the convention of api for other parts of the agent ?

Reply:
Because I'm introducing the Remote API powered by Remote Config next 😄